tensorrt_llm/_torch/visual_gen/executor.py

```
@@ -358,6 +358,12 @@ def run_diffusion_worker(
         device_id = rank % torch.cuda.device_count() if torch.cuda.is_available() else 0
         if torch.cuda.is_available():
             torch.cuda.set_device(device_id)
+            try:
+                from tensorrt_llm.llmapi.utils import configure_cpu_affinity
+
+                configure_cpu_affinity(device_id)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)